modules/tfcx/usermsg.cpp

```diff
@@ -120,10 +120,18 @@ void Client_Damage(void* mValue){
 
 	if ( FNullEnt( enemy ) )
 		break;
-
+	/*
+	ALERT(at_logged, "Clinet_damage Called with state-%d dmg-%d Netname'%s'(Class'%s') slno-'%d'\n",
+		mState, damage, STRING(mPlayer->pEdict->v.netname),
+		STRING(mPlayer->pEdict->v.classname), mPlayer->pEdict->serialnumber);
+
+	ALERT(at_logged, "   Called with enemy-slno-%d Netname:'%s'(Class'%s')\n", mPlayer->pEdict->serialnumber, 
+		STRING(enemy->v.netname), STRING(enemy->v.classname));
+	*/
```

Comment:
Please don't leave debugging lines in.

Reply:
I wished to keep them for future testing ? No?

Reply:
It's better to delete it. Otherwise it's clutter. If you want to log it with #if DEBUG or something, that's okay. But we don't want purely dead code in the tree.